aodndata/soop/soop_co2_nrt_nc_generator.py

```diff
@@ -291,7 +292,7 @@ def check_parameters(dataf, vessel_code, input_param, src_file):
         # var TYPE conversion to string outside this function
         for param in rt_input_parameters:
             if param not in set(['Type', 'PcDate', 'PcTime']):
-                dataf[param] = dataf[param].apply(pd.to_numeric, errors=coerce)
+                dataf[param] = dataf[param].apply(pd.to_numeric, errors='coerce')
```

Comment:
@bpasquer as discussed, this coerce function 1) appears that it should be a string and 2) breaks on Python 3

Reply:
@lbesnard are you able to have a look at this? 

It looks like the intention was to supply the string 'coerce' to the to_numeric method, but it had a reference to the coerce *function* supplied. 

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.to_numeric.html

Updating this to the string and running on Python 3, which has no builtin coerce function, the tests fail as follows:

```bash
======================================================================
FAIL: test_good_rt_aa_txt (test_aodndata.soop.test_soop_co2.TestSoopCo2Handler)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lwgordon/RubymineProjects/chef/src/python-aodndata/test_aodndata/soop/test_soop_co2.py", line 105, in test_good_rt_aa_txt
    check_params={'checks': ['cf', 'imos:1.4']})
  File "/home/lwgordon/RubymineProjects/chef/src/python-aodndata/.tox/py3/lib/python3.6/site-packages/aodncore/testlib/handlertest.py", line 39, in run_handler
    self.assertIsNone(handler.error)
AssertionError: ValueError("Variable 'TYPE' has dimensions ['TIME', 'string_10'] and shape (1087, 40), inconsistent with dimension sizes defined in template (1087, 10)",) is not None

======================================================================
FAIL: test_good_rt_in_txt (test_aodndata.soop.test_soop_co2.TestSoopCo2Handler)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lwgordon/RubymineProjects/chef/src/python-aodndata/test_aodndata/soop/test_soop_co2.py", line 84, in test_good_rt_in_txt
    check_params={'checks': ['cf', 'imos:1.4']})
  File "/home/lwgordon/RubymineProjects/chef/src/python-aodndata/.tox/py3/lib/python3.6/site-packages/aodncore/testlib/handlertest.py", line 39, in run_handler
    self.assertIsNone(handler.error)
AssertionError: ValueError("Variable 'TYPE' has dimensions ['TIME', 'string_10'] and shape (958, 40), inconsistent with dimension sizes defined in template (958, 10)",) is not None
```

This is the only thing in this package failing on Python 3, so would be good to get it sorted out!